pages/room/[id].js

```diff
@@ -75,6 +75,13 @@ export default function Room(props) {
     variables: { room: props.roomId },
   })
 
+  const roomSubscription = useSubscription(ROOM_SUBSCRIPTION, {
+    variables: { room: props.roomId },
+  })
+
+  //const [showHideCard] = useMutation(SHOW_HIDE_ROOM_CARD)
+  const [showHideCard] = useMutation(UPDATE_ROOM)
```

Comment:
```suggestion
  const [updateRoom] = useMutation(UPDATE_ROOM)
```

This should be more generic as we'll likely be using this mutation for other features